stm32-data-gen/src/chips.rs

```diff
@@ -481,8 +482,11 @@ impl PeriMatcher {
             ("STM32L.*:TIM(9|21|22):.*", ("timer", "v1", "TIM_2CH")),
             ("STM32L.*:TIM15:.*", ("timer", "v1", "TIM_2CH_CMP")),
             ("STM32L.*:TIM(16|17):.*", ("timer", "v1", "TIM_1CH_CMP")),
-            ("STM32L5.*:LPTIM.*:.*", ("lptim", "v2a", "LPTIM")),
-            ("STM32L.*:LPTIM(1|2|3):.*", ("lptim", "v1", "LPTIM")),
+            // LPTIM for STM32Lx
+            ("STM32L5.*:LPTIM.*:.*", ("lptim", "v1c", "LPTIM")),
+            ("STM32L4[PQR].*:LPTIM.*:.*", ("lptim", "v1b", "LPTIM")),
```

Comment:
L4S is also part of L4+, maybe it's missing here?

Reply:
whoops!